server/src/main/java/io/crate/replication/logical/MetadataTracker.java

```diff
@@ -213,9 +214,34 @@ private CompletableFuture<?> processSubscription(String subscriptionName) {
             subscription.connectionInfo().user()
         );
         CompletableFuture<Response> publicationsState = client.execute(PublicationsStateAction.INSTANCE, request);
-        CompletableFuture<Boolean> updatedClusterState = publicationsState.thenCompose(response ->
-            updateClusterState(subscriptionName, subscription, response)
-        );
+        CompletableFuture<Boolean> updatedClusterState = publicationsState.thenCompose(response -> {
+            // We cannot use replicationService.verifyTablesDoNotExist(subscriptionName, stateResponse)
+            // as it uses cluster metadata for comparison and will fail on the second round of every replication.
+            List<RelationName> existingTables = getExistingLocallyTables(subscription, subscriberState, response.tables());
+            if (!existingTables.isEmpty()) {
+                var msg = String.format(
+                    Locale.ENGLISH,
+                    "Subscription '%s' cannot be created, some relation(s) already exist. Check table pg_subscription_rel to see existing tables. " +
```

Comment:
Rephrased and also removed self subscription loop - there are at least 2 different cases which can lead to such state
1. create publication pub (t1) -> alter pub drop table t1 -> alter pub add table t1  (currently second step removes table but soon after changing behavior it will we caught by this check as well
2. create pub1 (t1) -> create pub (all tables)/ or create pub2 (t1,t2) -> whatever comes second to tracker will be caught by this message

So user should just be aware about existing table.